rho/authlistcommand.py

```diff
@@ -50,7 +50,7 @@ def _do_command(self):
         cred_list = vault.load_as_json(utilities.CREDENTIALS_PATH)
 
         for cred in cred_list:
-            if not cred.get('password') == '':
+            if cred.get('password') is not None:
```

Comment:
Suggestion: just display the password as `**********` unconditionally, even if it is None. Otherwise we're not really obscuring all passwords when we display them.